pump/storage/storage.go

```diff
@@ -736,15 +742,13 @@ func (a *Append) doGCTS(ts int64) {
 
 		if len(lastKey) > 0 {
 			irange.Start = lastKey
-			a.vlog.gcTS(decodeTSKey(lastKey))
 			doneGcTSGauge.Set(float64(oracle.ExtractPhysical(uint64(decodeTSKey(lastKey)))))
 		}
 		log.Info("has delete", zap.Int("delete num", deleteNum))
 	}
-
-	a.vlog.gcTS(ts)
+	wg.Wait()
 	doneGcTSGauge.Set(float64(oracle.ExtractPhysical(uint64(ts))))
-	log.Info("finish gc", zap.Int64("ts", ts), zap.Int("delete num", deleteNum))
+	log.Info("Finish GC", zap.Int64("ts", ts), zap.Int("delete num", deleteNum))
```

Comment:
it's bad to output this log if we break from `for loop`, it will make debug not clear enough